@xen-orchestra/web-core/lib/components/ui/dropdown-button/UiDropdownButton.vue

```diff
@@ -0,0 +1,103 @@
+<!-- v3 -->
+<template>
+  <button type="button" class="ui-dropdown-item" :class="{ selected: selected }" :disabled="isDisabled">
```

Comment:
You can use the shorthand here:
```suggestion
  <button type="button" class="ui-dropdown-item" :class="{ selected }" :disabled="isDisabled">
```